doc/user-guide/dask.rst

```diff
@@ -511,6 +511,13 @@ Notice that the 0-shaped sizes were not printed to screen. Since ``template`` ha
     mapped.identical(expected)
 
 
+.. tip::
+
+   As :py:func:`map_blocks` loads each block, reduce as much as possible objects consumed by user functions.
+   For example, drop useless variables before calling ``func`` with :py:func:`map_blocks`.
```

Comment:
Maybe slightly better: "loads each block into memory"

"load" has a specific meaning as the `.load()` method in xarrary, but that terminology may not be familiar to readers.

(This could also be updated in your proposed docstring edits, too)

Reply:
Good point, that makes it more clear!